ocs_ci/ocs/benchmark_operator.py

```diff
@@ -18,14 +18,12 @@
 
 # Local modules
 from ocs_ci.helpers import helpers
+from ocs_ci.ocs.constants import BMO_NAME
 from ocs_ci.ocs.exceptions import CommandFailed
 from ocs_ci.ocs.node import get_worker_nodes
 from ocs_ci.ocs.ocp import OCP, switch_to_default_rook_cluster_project
 from ocs_ci.utility.utils import TimeoutSampler
 
-# BMO stand for : BenchMark Operator
-# The benchmark operator name used for path / namespace etc.
-BMO_NAME = "benchmark-operator"
 # The benchmark operator git repository
```

Comment:
Any specific reason for moving this to constants?  There are many places in perf/e2e tests using this path already.

Reply:
Moved back to benchmark file